modules/navier_stokes/src/userobjects/RhieChowMassFlux.C

```diff
@@ -168,7 +169,8 @@ RhieChowMassFlux::setupCellVolumes()
   for (const auto & elem_info : _fe_problem.mesh().elemInfoVector())
   {
     const auto elem_dof = elem_info->dofIndices()[_global_pressure_system_number][0];
-    _cell_volumes->set(elem_dof, elem_info->volume() * elem_info->coordFactor());
+    if (elem_dof != libMesh::DofObject::invalid_id)
```

Comment:
that's for remote elements?

Reply:
Oh yes, I will try the `elem != remote_elem` comparison to make it nicer.

Reply:
Well, it is actually not that, I replaced with a more indicative check. I was just trying to optimize here.